src/types.ts

```diff
@@ -22,87 +19,40 @@ export type Failure = {
 
 export type SuccessOrFailure<T> = Success<T> | Failure
 
+export type PricingInOut = {
+  tokenInUsdPrice: BigNumber
+  tokenOutUsdPrice: BigNumber
+}
+
 /// -----------------------------------------------------------------------
 /// Swap Path
 /// -----------------------------------------------------------------------
 
-export type SwapPath = {
-  swapRouter: string
-  swapType: SwapType
-  path: string[]
-  amountOutMin: BigNumber
-  //Extra vars
-  amountOut: BigNumber
-}
+export type SwapPath = ISoulZap_UniV2.SwapPathStruct
 
 /// -----------------------------------------------------------------------
 /// Liquidity Path
 /// -----------------------------------------------------------------------
 
-export type LiquidityPath = {
-  lpRouter: string
-  lpType: LPType
-  amountAMin: BigNumber
-  amountBMin: BigNumber
-  lpAmount: BigNumber
-}
+export type LiquidityPath = ISoulZap_UniV2.LiquidityPathStruct
 
 /// -----------------------------------------------------------------------
 /// Swap Params
 /// -----------------------------------------------------------------------
 
-export type SwapParams = {
-  tokenIn: string
-  amountIn: BigNumber
-  tokenOut: string
-  path: SwapPath
-  to: string
-  deadline: BigNumber
-}
-
-export type SwapData = {
-  encodedTx: string
-  swapParams: SwapParams
-  feeSwapPath: SwapPath
-  priceImpactPercentages: BigNumber[]
-}
-
-export type SwapDataExtras = {
```

Comment:
Just named it 'extra' in case we were going to add more things outside the original swapdata besides pricing. But call it PricingInOut works for now as that's all we have for now